test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Microsoft.VisualStudio.LanguageServices.Razor.Test.csproj

```diff
@@ -3,6 +3,10 @@
   <PropertyGroup>
     <TargetFramework>net461</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
+    
+    <!-- The binding redirects are needed due to a conflict between the test host and Roslyn regarding System.Collections.Immutable -->
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
The test SDK is supposed to generate this for you though: https://github.com/Microsoft/vstest/blob/6924402e13817bd2c749d6ff5027db62ca61225c/src/package/nuspec/Microsoft.NET.Test.Sdk.targets#L51.